syntax = "proto3";
package nezha.kubeproxy;

import "google/api/annotations.proto";

// The greeting service definition.
service Greeter {
    // Sends a greeting
    rpc SayHello (HelloRequest) returns (HelloReply) {
        // option (google.api.http) = {
        //     post: "/v3/kv/range"
        //     body: "*"
        // };
    }
  }
  
// The request message containing the user's name.
message HelloRequest {
    string name = 1;
}
  
// The response message containing the greetings
message HelloReply {    
    string message = 1;
}


